spec/schema.json

```diff
@@ -118,6 +118,29 @@
           "type": "string"
         }
       ],
+      "get": {
+        "summary": "Get all attachments from the Objection",
+        "produces": [
+          "application/json"
+        ],
+        "tags": [
+          "Strike Off Objections"
+        ],
+        "responses": {
+          "200": {
+            "description": "Attachments returned successfully",
+            "schema": {
+              "type": "array",
+              "items": {
+                "$ref": "#/definitions/Attachment"
+              }
+            }
+          },
+          "404": {
+            "description": "the objection or attachments were not found"
```

Comment:
This to me suggests that if an objection has no attatchments, it will return a 404.
Would returning a 200 with an empty array be clearer?

Reply:
it does look better, i'll update